DOCUMENTATION.md

```diff
@@ -63,7 +72,7 @@ Checks if data has been received in the child thread. If data was received it wi
 
 ## AwaitData
 
-Waits untill data is received from the server. It is best practise to call this function after ReceiveDataAsync().
+Waits untill data is received from the server. If this function is called without ReceiveDataAsync() the main thread will wait until data is received on the socket. Otherwise it will wait until the sub thread returned data.
```

Comment:
until, not untill

Reply:
By "called without ReceiveDataAsync()", do you mean "called before calling ReceiveDataAsync()"? The current phrasing indicates ReceiveDataAsync() would be passed as a parameter.